src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -264,13 +273,13 @@ private void HandleLocalResource(OpenDocumentationLinkEventArgs e)
             this.shouldLoadDefaultContent = false;
         }
 
-        private string GetGraphLinkFromMDLocation(Uri link)
+        private string GetGraphLinkFromMDLocation(Uri link,bool isOwnedByPackage)
```

Comment:
Just for curiosity you already defined the property IsOwnedByPackage then why passing it as a parameter to a method?

Reply:
Good eye, and this is very important because unfortunately the singleton nature of the Docs browser model makes it very easy to have data which has not yet been set correctly.

For example before I made this change it was possible for me to reference `isOwnedByPackage` before it had event been calculated for the node docs request we were parsing. By using the param as a dependency it's clear to callers that the method will be using it and it should be up to date.

Reply:
Is it worth making it a class member in that case?